tests/test_convert.py

```diff
@@ -51,6 +51,7 @@ def test_format_atkinson_precincts(atkinson_precincts):
     assert len(results.keys()) == 25
     assert "President of the United States" in results.keys()
     assert results["President of the United States"]["precinctsReportingPct"] == 100
+    assert results["President of the United States"]["timestamp"] == "2020-11-06 13:05:50-05:00"
```

Comment:
We need this to be in the format `2020-11-06T13:05:50Z`

Reply:
should be better now, but you should probably double check the logic: https://github.com/washingtonpost/elex-clarity/pull/6/files#diff-de97c87d907e2bf3af5b2030985c1683dc38eb2b101ce532a007fc5bd6247ff5R143